### PR TITLE
[MIG] base_vat: Nothing to do

### PR DESCRIPTION
**Pull request by cubells**
Nothing to do

cc @Tecnativa

### CODE REVIEW

File: addons/base_vat/migrations/11.0.1.0/openupgrade_analysis.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 NEW ir.ui.view: base_vat.res_config_settings_view_form
 NEW ir.ui.view: base_vat.view_partner_short_form
 DEL ir.ui.view: base_vat.company_form_vat
+# Nothing to do
```
Comment (pedrobaeza) [inline addons/base_vat/migrations/11.0.1.0/openupgrade_analysis.txt]: You should create a new `openupgrade_analysis_work.txt` file, not modifying this one.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
@cubells I see you addressed @pedrobaeza's comment. That's great ;-) You should also put down a comment about it so that this PR shows up in Pedro's notifications. That way, Pedro has a way of knowing that this PR can be reassessed.

**Comment by StefanRijnhart** [comment]
I'll merge as it is easy for me to validate on Pedro's behalve in this trivial case.

**Comment by pedrobaeza** [comment]
Thanks

**Pull request merged**